src/bin/stratis-min/stratis-min.rs

```diff
@@ -42,18 +41,8 @@ fn parse_args() -> Command {
                 Command::new("start")
                     .arg(Arg::new("id").required(true))
                     .arg(Arg::new("name").long("name").num_args(0))
-                    .arg(
-                        Arg::new("unlock_method")
-                            .long("unlock-method")
-                            .num_args(1)
-                            .value_parser(PossibleValuesParser::new(UnlockMethod::VARIANTS)),
-                    )
-                    .arg(
-                        Arg::new("prompt")
-                            .long("prompt")
-                            .num_args(0)
-                            .requires("unlock_method"),
-                    ),
+                    .arg(Arg::new("token_slot").long("token_slot").num_args(1))
```

Comment:
For ```token_slot``` arguments should be able to use ```.value_parser(clap::value_parser!(u32))``` in the parser, i.e., here, and then drop the extra handling when you process the parsed arg.